packages/devtools_app/lib/src/inspector/flutter/inspector_service_flutter_extension.dart

```diff
@@ -69,6 +69,9 @@ extension InspectorFlutterService on ObjectGroup {
       if (root == null) {
         return null;
       }
+      String toStringAsFixed(double num) {
+        return num.toStringAsFixed(3);
+      }
```

Comment:
I changed this because I encountered some precision error with our algorithm to check overflow (there's one case where we get the child widget width is 136.6 and the parent is 136.5 but it's not really overflowing in the real Flutter app)

Reply:
I've seen this too.  But why is the change going in the string representation and not the number comparison?

Reply:
Discussed offline and switched to using an epsilon for the num comparisons